datasets/index.md

```diff
@@ -1,9 +1,8 @@
 ---
 layout: default
 title: Data Sources
-nav_order: 110
```

Comment:
Can you revert this one? Or is it auto-generated?

Reply:
This was autogenerated. Should i revert?

Reply:
yes, please. I should check why it changes this file...

Reply:
reverted this file